src/components/loader/loader-bar.style.js

```diff
@@ -0,0 +1,65 @@
+import styled, { css, keyframes } from "styled-components";
+import PropTypes from "prop-types";
+import baseTheme from "../../style/themes/base";
+import { LOADER_BAR_SIZES } from "./loader-bar.config";
+
+const INNER_BAR_LENGTH = "128px";
+const OUTER_BAR_LENGTH = "256px";
+
+const innerBarAnimation = keyframes`
+  0% {
+    left: -${INNER_BAR_LENGTH}
+  }
+  100% {
+    left: ${OUTER_BAR_LENGTH}
+  }
+`;
+
+const StyledLoaderBar = styled.div`
+  ${({ size, theme }) => css`
+    display: inline-block;
+    height: ${getHeight(size)};
+    width: ${OUTER_BAR_LENGTH};
+    background-color: ${theme.colors.loadingBarBackground};
+    overflow: hidden;
+    position: relative;
+  `}
+`;
+
+const InnerBar = styled.div`
+  ${({ theme, size }) => css`
+    position: absolute;
+    background-color:  ${theme.colors.primary};
+    width: ${INNER_BAR_LENGTH};
+    height: ${getHeight(size)}
```

Comment:
missing `;`, v4 of `styled-components` automatically adds them in but I believe that was taken out in v5 so we'll have an issue when we upgrade